src/main/java/fr/moribus/imageonmap/commands/maptool/ExploreCommand.java

```diff
@@ -55,31 +56,35 @@
 public class ExploreCommand extends IoMCommand {
     @Override
     protected void run() throws CommandException {
-        if (args.length < 1) {
-            Gui.open(playerSender(), new MapListGui(playerSender()));
-        } else {
-            if (Permissions.LISTOTHER.grantedTo(sender)) {
-                String name = args[0];
-                Player sender = playerSender();
-                offlinePlayerParameter(0, uuid -> {
-                    if (uuid == null) {
-                        try {
-                            throwInvalidArgument(I.t("Player not found."));
-                        } catch (CommandException e) {
-                            PluginLogger.error("CommandException " + e);
-                            return;
-                        }
-                    }
+        ArrayList<String> arguments = getArgs();
+        if (arguments.size() > 1) {
+            warning(I.t("Too many parameters! Usage: /maptool explore [playername]"));
```

Comment:
Aussi ajouter l'usage à `@CommandInfo` (idem que plus haut je ne peux le suggérer).
```suggestion
            throwInvalidArgument(I.t("Too many parameters!"));
```

Reply:
C'est le cas pour presque toutes les commandes